src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTrackerGeneratorInfo.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis;
+
+internal partial class SolutionCompilationState
```

Comment:
nit: should there be a '.' in the file name to match the other one?

Reply:
I've left that out because otherwise i can't see the full name in explorer :)